RxSwift/Traits/PrimitiveSequence.swift

```diff
@@ -124,7 +124,31 @@ extension PrimitiveSequenceType where TraitType == SingleTrait {
             case .error(let error):
                 observer(.error(error))
             case .completed:
-                rxFatalError("SingleProtocol")
+                rxFatalError("Singles can't emit a completion event")
+            }
+        }
+    }
+
+    /**
+     Subscribes a success handler, and an error handler this Single.
```

Comment:
I also dabbled with that thought. I'm not 100% sure. 
Single/Completable/Maybe won't actually ever be a sequence, they only emit a single event and terminate, right? They are quite unite "beasts" of themselves providing a sub-section of what a full Raw Observable is. 

That's why I thought it would be nice to refer to it as "Subscribes a success handler for this Single (which is actually a subset of a Sequence)" , so it's more clear it's referring to a Single. But I really don't have any specific preference. 

Do you prefer changing all of them to "sequence" like you suggested to keep a uniform language?